### PR TITLE
Prepare hashcode

### PR DESCRIPTION
**Pull request by kindermoumoute**

### CODE REVIEW

File: .idea/ElCredo.iml [hunk 1]
```diff
@@ -1,10 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (rb62680) [inline .idea/ElCredo.iml:1]: @kindermoumoute Why did you remove this?

File: encoding.go [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+	"fmt"
+	"io/ioutil"
+	"strconv"
+	"strings"
+)
+
+func read() Pizza {
+	b, err := ioutil.ReadFile("input/" + fileName + ".in")
+	if err != nil {
+		panic(err)
```
Comment (rb62680) [inline encoding.go:15]: @kindermoumoute Do we need to exit here?

File: training/pizza/main.go [hunk 3]
```diff
@@ -1,13 +1,116 @@
...
+func (p *Pizza) print(m map[int]*Cell) {
+	fmt.Printf("\n\nPIZZA INITIALE")
+	letter := []string{"T", "M"}
+	for i, c := range p.Cell {
+		if i%p.Rows == 0 {
+			fmt.Printf("\n")
+		}
+		fmt.Print(letter[c.Ingredient])
+	}
+	fmt.Printf("\nPIZZA DECOUPEE")
```
Comment (rb62680) [inline training/pizza/main.go:55]: @kindermoumoute Please change this to use english only.

**Review by rb62680 — CHANGES_REQUESTED** [review]
Made some suggestions.

File: training/pizza/split_slice.go [hunk 4]
```diff
@@ -1 +1,31 @@
...
+	sizes := make([]size, 0, int(sqrt))
+	for i := 1; float64(i) <= sqrt; i++ {
+		if h%i == 0 && i <= max {
+			sizes = append(sizes, size{row: i, column: h / i})
+		}
+	}
+	return sizes
+}
+
+//func (p *Pizza) splitPizza() {
```
Comment (maxencehenneron) [inline training/pizza/split_slice.go:20]: please remove this if you don't use it
